servers/rendering/renderer_rd/forward_clustered/render_forward_clustered.cpp

```diff
@@ -944,25 +944,33 @@ void RenderForwardClustered::_fill_render_list(RenderListType p_render_list, con
 			// LOD
 
 			if (p_render_data->scene_data->screen_mesh_lod_threshold > 0.0 && mesh_storage->mesh_surface_has_lod(surf->surface)) {
-				// Get the LOD support points on the mesh AABB.
-				Vector3 lod_support_min = inst->transformed_aabb.get_support(p_render_data->scene_data->cam_transform.basis.get_column(Vector3::AXIS_Z));
-				Vector3 lod_support_max = inst->transformed_aabb.get_support(-p_render_data->scene_data->cam_transform.basis.get_column(Vector3::AXIS_Z));
-
-				// Get the distances to those points on the AABB from the camera origin.
-				float distance_min = (float)p_render_data->scene_data->cam_transform.origin.distance_to(lod_support_min);
-				float distance_max = (float)p_render_data->scene_data->cam_transform.origin.distance_to(lod_support_max);
-
 				float distance = 0.0;
 
-				if (distance_min * distance_max < 0.0) {
-					//crossing plane
-					distance = 0.0;
-				} else if (distance_min >= 0.0) {
-					distance = distance_min;
-				} else if (distance_max <= 0.0) {
-					distance = -distance_max;
+				// Check if camera is NOT inside the mesh AABB.
+				Vector3 camera_position = p_render_data->scene_data->cam_transform.origin;
+				Vector3 aabb_start = inst->transformed_aabb.get_position();
+				Vector3 aabb_end = inst->transformed_aabb.get_end();
+
+				if ((aabb_start.x <= camera_position.x && camera_position.x <= aabb_end.x &&
+							aabb_start.y <= camera_position.y && camera_position.y <= aabb_end.y &&
+							aabb_start.z <= camera_position.z && camera_position.z <= aabb_end.z) == false) {
```

Comment:
```suggestion
				if (!inst->transformed_aabb.has_point(camera_position)) {
```

Reply:
I don't see any difference between the if statement you wrote and the `AABB::has_point()` function. Is there something I'm missing here?

Reply:
Yes you are right, it is the same. At the time i didn't knew AABB had the has_point function.